Packs/SplunkPy/Integrations/SplunkPy/SplunkPy_description.md

```diff
@@ -1 +1,3 @@
 Use the SplunkPy integration to fetch incidents from Splunk ES, and query results by SID.
+
+To use Splunk token authentication, enter the text: `_token` to the **Username** field and your token value to the **Password** field.
```

Comment:
maybe you should add an explanation about how to get the token from splunk's side?